wagtailio/project_styleguide/templates/patterns/base.html

```diff
@@ -0,0 +1,45 @@
+{% load static wagtailuserbar %}
+
+<!DOCTYPE html>
+<html class="no-js" lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <title>
+            {% block title %}
+                {% if self.seo_title %}{{ self.seo_title }}{% else %}{{ self.title }}{% endif %}
+            {% endblock %}
+            {% block title_suffix %}
+                {% with self.get_site.site_name as site_name %}
+                    {% if site_name %}- {{ site_name }}{% endif %}
+                {% endwith %}
+            {% endblock %}
+        </title>
+        <meta name="description" content="" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+        {# Global stylesheets #}
+        <link rel="stylesheet" href="{% static 'css/main.css' %}">
```

Comment:
we switched this to use manifest loader https://github.com/wagtail/wagtail.org/pull/175/commits/277de0fa91e559bbf3d693e3a7fac91b6e599000

`<link rel="stylesheet" href="{% manifest 'css/main.css' %}">`

You'll need to add the `manifest` tag as well

Reply:
Rats, you are right! good catch.